"Incomplete: No specs found" when running tests

I cloned the project, then `npm install` and `npm run test`.
Got output:
```
➜  opentelemetry-plugin-mongoose git:(master) npm run test

> @wdalmut/opentelemetry-plugin-mongoose@0.0.28 test /Users/amirblum/repos/opentelemetry-plugin-mongoose
> node --no-deprecation ./node_modules/jasmine/bin/jasmine.js

Randomized with seed 96266
Started


No specs found
Finished in 0.003 seconds
Incomplete: No specs found
Randomized with seed 96266 (jasmine --random=true --seed=96266)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @wdalmut/opentelemetry-plugin-mongoose@0.0.28 test: `node --no-deprecation ./node_modules/jasmine/bin/jasmine.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @wdalmut/opentelemetry-plugin-mongoose@0.0.28 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/amirblum/.npm/_logs/2020-05-20T15_30_26_325Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,8 @@
     "build": "./node_modules/.bin/tsc",
     "watch": "npm run build -- -w",
     "coverage": "./node_modules/.bin/nyc npm test",
-    "report": "./node_modules/.bin/nyc report --reporter=text-lcov > coverage.lcov"
+    "report": "./node_modules/.bin/nyc report --reporter=text-lcov > coverage.lcov",
+    "postinstall": "npm run build"
   },
   "files": [
     "lib/**/*.js",

